### DISCUSSION TITLE
How I can get Registrations for a Competition from SQL dump?

### DISCUSSION THREAD

**Discussion by castus** [thewca/worldcubeassociation.org#8709] [Q&A]
How I can get Registrations for a Competition from SQL dump? AFAIK there is no `Registrations` table in SQL dump, right? Is this on purpose? If so, how get that data other than scraping HTML?

**Comment by castus** [comment]
Any comments?

**Comment by gregorbg** [comment]
Hey Krzysztof,

Sorry for the delayed response, we are not used to receiving support requests through this GitHub Discussions feature.
The registration information is available only in the "Developer Export", not in the more light-weight "Results Export".

What is your use-case / what feature are you trying to build? Depending on your specific case, you might consider using the so-called "WCIF" output for any competition, which is available through a JSON REST API and it includes the registrations.
